tools/scripts/build/build-mobile-curriculum.js

```diff
@@ -0,0 +1,54 @@
+const fs = require('fs');
+const path = require('path');
+
+exports.buildMobileCurriculum = function buildMobileCurriculum(json) {
+  const mobileStaticPath = path.resolve(__dirname, '../../../client/static');
+
+  if (!fs.existsSync(`${mobileStaticPath}/mobile`)) {
+    fs.mkdir(`${mobileStaticPath}/mobile`, err => {
+      return err;
+    });
```

Comment:
```suggestion
    fs.mkdirSync(`${mobileStaticPath}/mobile`)
```
It needs to be `mkdirSync` or there will be a race condition between `mkdir` and `writeJson`.  A race that `mkdir` will almost certainly win, but still!

Also, throwing an error is fine here: if this doesn't work I doubt we can recover.